### ISSUE TITLE
Contact with role editor cannot edit data resource

### ISSUE THREAD

**Issue by charvolant** [AtlasOfLivingAustralia/collectory#158]
A contact was given editor role on https://collections.ala.org.au/public/show/dr413 but attempting to edit the description results in

```
Error 500: Cannot invoke method _get() on null object
Servlet: grailsDispatcherServlet
URI: /dataResource/edit/dr413
Exception Message: Cannot invoke method _get() on null object
Caused by: Cannot invoke method _get() on null object
Class: CooperatingFilterWrapper
At Line: [31]
```

Stack trace

```
java.lang.NullPointerException: Cannot invoke method _get() on null object

         at au.org.ala.collectory.ProviderGroupController.isAuthorisedToEdit(ProviderGroupController.groovy:861)

         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

         at au.org.ala.collectory.ProviderGroupController$_closure6.doCall(ProviderGroupController.groovy:90)

         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

         at org.grails.core.DefaultGrailsControllerClass$MethodHandleInvoker.invoke(DefaultGrailsControllerClass.java:223)

         at org.grails.core.DefaultGrailsControllerClass.invoke(DefaultGrailsControllerClass.java:188)

         at org.grails.web.mapping.mvc.UrlMappingsInfoHandlerAdapter.handle(UrlMappingsInfoHandlerAdapter.groovy:90)

         at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1043)
```

**Comment by sughics** [comment]
PR with fix: https://github.com/AtlasOfLivingAustralia/collectory/pull/177

**Comment by sughics** [comment]
Fix Release in v 3.1.2
